scripts/run-e2e-tests.sh

```diff
@@ -55,11 +124,12 @@ function install_cluster() {
   k8s_cluster_name=$1
   kind_node_image=$2
 
-  # install ginkgo and gomega for e2e tests
+  # Check if go is installed.
   check_cmd "go"
+
+  # install ginkgo and gomega for e2e tests
   go get -v github.com/onsi/ginkgo/ginkgo
   go get -v github.com/onsi/gomega
-  export PATH=$PATH:$HOME/gopath/bin
   check_cmd "ginkgo"
```

Comment:
I have one minor thing left: should we add a comment that the install for ginkgo can be found in the directory: ``echo `go env GOPATH`/bin``

Reply:
Should be good now. :)